### ISSUE TITLE
Solve text overflow in github cards

### ISSUE THREAD

**Issue by bregydoc** [minskylab/supersense#24]
![image](https://user-images.githubusercontent.com/18302365/93945114-ec51dc00-fcfb-11ea-9da3-d82a46a54658.png)

As you can see above, the issue is visual and in order to solve, you need to update the card component and maybe the grid layout. The observer implementation is [here](https://github.com/minskylab/supersense/tree/master/observer). Finally, you can use `yarn dev` to launch the only frontend, but I recommend use [task](https://taskfile.dev) in the root. With `task start` command, you can perform all the compilation routine and launch the complete suite of supersense, only remember your .env config with your config.

**Comment by cdikibo** [comment]
This looks like CSS problem. Has anyone else called dibs on this issue?

**Comment by bregydoc** [comment]
Yes, the issue is related to the CSS (abstracted with [React](https://reactjs.org/) and [ChakraUI](https://next.chakra-ui.com/)), specifically in the [EventCard Component](https://github.com/minskylab/supersense/blob/master/observer/components/atoms/eventCard/index.tsx).
